### DISCUSSION TITLE
How to enable resizable column for table in sun editor.

### DISCUSSION THREAD

**Discussion by Albinbritto** [JiHong88/SunEditor#1325] [Q&A]
Hi,

   Anyone please guide me how to enable resizable column for table.

**Comment by JiHong88** [comment]
It will be added in version 3.0.0.

**Reply by Albinbritto** [reply]
thanks for the reply.I have one requirement,I need to apply font size ,font family, bold, italic etc only for the selected cell in table.whether it is possible now?
